### ISSUE TITLE
Searching in OU returns empty items array

### ISSUE THREAD

**Issue by ChinwalPrasad** [Adldap2/Adldap2-Laravel#707]
- Laravel Version: 5.7
- Adldap2-Laravel Version: 6.0.2
- PHP Version: 7.1
- LDAP Type: ActiveDirectory 

### Description:
Goal: TO retrieve all the users in OU.
Steps: 
```
$users = Adldap::search()->ous()->find('Company Users');
$members = Adldap::search()->in($users)->get();
dd($members);
```
this returns an empty array of items and sometimes just returns false.

FYI: The statement ```$users = Adldap::search()->ous()->find('Company Users');``` returns me the OU.

These steps where mentioned [here](https://github.com/Adldap2/Adldap2-Laravel/issues/389)

Also could you link the documentation to move the user to different OU and to delete a user in the OU? 

Thanks for the help.

**Comment by stevebauman** [comment]
Hi @ChinwalPrasad,

This actually appears to be a bug, I tested this on my end too. I'm surprised I haven't run into this use case.

I discovered that an LDAP filter is absolutely mandatory for any LDAP results to be returned - otherwise an LDAP exception is thrown.

Right now you can simply use this:

```php
$users = Adldap::search()->ous()->find('Company Users');

$members = Adldap::search()->in($users)->whereHas('objectclass')->get();

dd($members);
```

Patching this now though, will be a new release shortly. Thanks!

**Comment by ChinwalPrasad** [comment]
@stevebauman 

**Thank you so much for the quick reply. Appreciate it.**

I'll use your solution for now and wait for the patch.
Could you also help in locating the documentation to 
- Move the user to different OU.
- Delete the user from the OU.

Thanks.

**Comment by stevebauman** [comment]
Hi @ChinwalPrasad, no problem!

Also, of course, here's the [link](https://adldap2.github.io/Adldap2/#/models/model?id=moving-renaming).

```php
$ou = Adldap::search()->ous()->find('Company Users');

$user = Adldap::search()->users()->find('jdoe');

$user->move($ou);
```

**Comment by ChinwalPrasad** [comment]
Amazing @stevebauman . Thanks again. 

Your Package is the best  🥇

**Comment by stevebauman** [comment]
Thanks so much for your kind words @ChinwalPrasad! That really means a lot to me :heart:

And no problem at all :smile:

**Comment by stevebauman** [comment]
Fixed in [v10.0.8](https://github.com/Adldap2/Adldap2/releases/tag/v10.0.8). Thanks again for the report!
